packages/devtools_app/lib/src/shared/config_specific/file/_file_web.dart

```diff
@@ -21,12 +21,16 @@ class FileSystemWeb implements FileIO {
   String exportDirectoryName({bool isMemory = false}) => 'LocalStorage';
 
   @override
-  void writeStringToFile(
+  void writeContentsToFile<T>(
     String filename,
-    String contents, {
+    T contents, {
     bool isMemory = false,
   }) {
-    _files.putIfAbsent(filename, () => contents);
+    if (contents is String) {
```

Comment:
should this be `if (contents is String || contents is Uint8List)`?

Reply:
Nope. For web we use different mechanism for Uint8List, merged earlier today.
I am not even sure that we use _file_web.dart even for String. Most likely, no.